app_remote/flaskr/static/js/piiAppClient.js

```diff
@@ -0,0 +1,20 @@
+class PiiAppClient {
```

Comment:
I realise this is not a big page, so feel free to ignore

- It's good practice to add name spacing to front-end classes and functions, as the js gets loaded into global scope when loaded into a page. Not using namespaces increases likelihood of collisions of names which causes weird errors
- Not likely to happen here, however including comment for completeness

Reply:
That's a good point! Assigned the class to a constant.